transmission_interface/include/transmission_interface/differential_transmission.h

```diff
@@ -264,6 +281,38 @@ inline void DifferentialTransmission::actuatorToJointPosition(const ActuatorData
   *jnt_data.position[1] = (*act_data.position[0] / ar[0] - *act_data.position[1] / ar[1]) / (2.0 * jr[1]) + jnt_offset_[1];
 }
 
+inline void DifferentialTransmission::actuatorToJointAbsolutePosition(const ActuatorData& act_data,
+                                                                    JointData&    jnt_data)
+{
+  assert(numActuators() == act_data.absolute_position.size() && numJoints() == jnt_data.absolute_position.size());
+  assert(act_data.position[0] && act_data.absolute_position[1] && jnt_data.absolute_position[0] && jnt_data.absolute_position[1]);
+
+  std::vector<double>& ar = act_reduction_;
+  std::vector<double>& jr = jnt_reduction_;
```

Comment:
Should be at least `const`. Not sure what this project's preferences are on things like `auto`, but could consider `const auto&`.

Reply:
Just realized that these are simply mimicking the pattern in the exsiting functions. They should be const both here and there, though perhaps that's better as a follow-up PR

Reply:
I've added consts to the other functions too